model/Shop/ShopUbytovani.php

```diff
@@ -236,7 +236,8 @@ public static function ulozObjednaneUbytovaniUcastnika(
     private            $pnDny                = 'shopUbytovaniDny';
     private            $pnPokoj              = 'shopUbytovaniPokoj';
     private            $pnCovidFreePotvrzeni = 'shopCovidFreePotvrzeni';
-    private bool $ubytovatZbyle = false;
+    private bool       $omezitNaSpacáky      = true; // přepínač pro omezení běžných návštěvníků čistě na spacáky
```

Comment:
Diakritiku v názvech proměnných mi zavedl Roman pro Preact a byť mi to vždycky zvedne koutky, tak v PHP bych se držel původní `cestiny`.

Reply:
Opraveno. Rovnou jsem se objektově rozvášnil.